jetty-server/src/main/config/etc/jetty-gzip.xml

```diff
@@ -18,8 +18,8 @@
         <Set name="dispatcherTypes" property="jetty.gzip.dispatcherTypes"/>
         <Set name="includedMethodList" property="jetty.gzip.includedMethodList"/>
         <Set name="excludedMethodList" property="jetty.gzip.excludedMethodList"/>
-        <Set name="includedMimeTypes" property="jetty.gzip.includedMimeTypeList"/>
-        <Set name="excludedMimeTypes" property="jetty.gzip.excludedMimeTypeList"/>
```

Comment:
If these methods did not exist, then the changes from #6398 should have detected that???

Reply:
Ah they did exist, they just had the wrong signature.